whois-api/src/test/java/net/ripe/db/whois/api/log/JettyRequestLogTestIntegration.java

```diff
@@ -152,11 +178,23 @@ public void password_filtered_case_insensitive() throws Exception {
                 .get(WhoisResources.class);
 
 
-        String actual = fileToString(getRequestLogFilename());
+        String actual = fileToString(getRequestLog());
         assertThat(actual.toLowerCase(), containsString("password=FILTERED".toLowerCase()));
         assertThat(actual, not(containsString("pass1")));
     }
 
+    @Test
+    public void override_filtered_case_insensitive() throws Exception {
+        RestTest.target(getPort(), "whois/test/person/TP1-TEST?oVeRrIdE=overrideUser,overPASS1")
+                .request()
+                .get(WhoisResources.class);
+
+
+        String actual = fileToString(getRequestLog());
+        assertThat(actual.toLowerCase(), containsString("override=overrideUser,FILTERED".toLowerCase()));
```

Comment:
Should we test that the query parameter name has not changed (as in, match on `oVeRrIdE`). This allows us to detect if a user is not using all lowercase in a query param name.

Reply:
yes, i think we should. i copied this test from the "PassWord" test but now you mention it, i don't think it's a good idea to 'lowerCase' everything in the test. updated in 44045027c